spec/models/manageiq/providers/vmware/infra_manager/vm_spec.rb

```diff
@@ -43,14 +38,35 @@
   end
 
   context "supports_clone?" do
-    let(:ems)  { FactoryGirl.create(:ems_vmware) }
-    let(:vm)   { FactoryGirl.create(:vm_vmware, :ext_management_system => ems) }
-
     it "returns true" do
       expect(vm.supports?(:clone)).to eq(true)
     end
   end
 
+  context("supports_shutdown_guest?") do
+    let(:vm_status) { {:raw_power_state => power_state_suspended, :tools_status => 'toolsNotInstalled'} }
```

Comment:
I set out to maintain the original style of `shared_examples`. Also it was handling both `supports?` and `is_avail`. 

But I since further deviated from that approach.  So it makes sense to remove this method now.  See my latest update. 

Thanks